packages/gitbeaker-requester-utils/src/RequesterUtils.ts

```diff
@@ -50,17 +50,17 @@ export function defaultRequest(
   service: DefaultRequestService,
   { body, query, sudo, method = 'get' }: DefaultRequestOptions = {},
 ): DefaultRequestReturn {
-  const { headers } = service;
+  const { headers, additionalBody } = service;
   let bod: FormData | string;
 
   if (sudo) headers.sudo = sudo;
 
   // FIXME: Not the best comparison, but...it will have to do for now.
   if (typeof body === 'object' && body.constructor.name !== 'FormData') {
-    bod = JSON.stringify(decamelizeKeys(body));
+    bod = JSON.stringify(decamelizeKeys({ ...body, ...additionalBody }));
     headers['content-type'] = 'application/json';
   } else {
-    bod = body as FormData;
+    bod = { ...body, ...additionalBody } as FormData;
```

Comment:
Nevermind - this addition is bad and results in the following error (I was able to isolate it and can confirm that it's caused precisely by this; reverting to the previous version fixes the error).

```
TypeError: Failed to construct 'Request': Request with GET/HEAD method cannot have body.
    at new Ky (content.js:489)
    at ky (content.js:724)
    at content.js:1611
    at Generator.next (<anonymous>)
    at content.js:1558
    at new Promise (<anonymous>)
    at ./gitbeaker/packages/gitbeaker-browser/src/KyRequester.ts.__awaiter (content.js:1554)
    at handler (content.js:1603)
    at Object.requester.<computed> [as get] (content.js:5376)
    at content.js:2093
```

Reply:
Then well have to think of another way around this since sending FormData would break this feature.